docs/hacking.md

```diff
@@ -66,15 +66,15 @@ Of course, as for `npm test`, you can explictely pass the pass to the Kinto serv
 
     $ npm run dist
 
-This should have created the following assets:
+This should have created the following assets, with the current version in the filename (here, `1.0.0`):
```

Comment:
Maybe rephrase 'here' to 'for instance', so that we don't have to update this file with each release.